pkg/local_object_storage/engine/evacuate.go

```diff
@@ -164,6 +164,8 @@ mainLoop:
 						}
 						continue loop
 					}
+
+					e.log.Debug("could not put to shard, trying another", zap.String("shard", shards[j].ID().String()))
```

Comment:
if last shard failed there wont be more `trying`

Reply:
not a problem to me, classic iteration: "try another shard; shard list is over; i finished cycle"

Reply:
what exactly would you expect here?